app/src/debug/java/org/simple/clinic/DebugClinicApp.kt

```diff
@@ -114,7 +114,7 @@ class DebugClinicApp : ClinicApp() {
         .patientModule(object : PatientModule() {
           override fun providePatientConfig(): Observable<PatientConfig> {
             return super.providePatientConfig()
-                .map { it.copy(scanSimpleCardFeatureEnabled = true) }
+                .map { it.copy(scanSimpleCardFeatureEnabled = true, showRecentPatients = true) }
```

Comment:
We should keep this flag disabled everywhere until the development finishes.
Hence, this should be removed.